collect_app/src/main/java/org/odk/collect/android/utilities/ApplicationConstants.java

```diff
@@ -96,6 +96,7 @@ private ApplicationConstants() {
         public static final int GEOSHAPE_CAPTURE = 20;
         public static final int GEOTRACE_CAPTURE = 21;
         public static final int ARBITRARY_FILE_CHOOSER = 22;
+        public static final int FONT_SIZE_MATCHER = 23;
```

Comment:
Not sure about the name of this. Usually it's nice to have these express "what has happened". Could it be `FONT_SIZE_CHANGED`?

Reply:
As with my above comment this should be for any setting change so maybe this should be `PREFERENCES` or something similar.

Reply:
Ok, I will resolve that.

Reply:
I didn't create a feature branch here for the issue. So will I have to close this PR and create a new one from the feature branch?

Reply:
Yeha I think so. As far as I know there isn't a way to edit the branch of a PR.